core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -491,9 +491,27 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
                       <strong><[thread.authorSecondLastMessage | truncate:7]>, </strong>
                     </span>
                   </span>
-                  <span ng-if="thread.lastMessageRead"><[thread.authorLastMessage | truncate:7]></span>
-                  <span ng-if="!thread.lastMessageRead">
-                    <strong><[thread.authorLastMessage | truncate:7]></strong>
+                  <span ng-if="!thread.authorSecondLastMessage">
```

Comment:
FYI, I find the current logic really confusing and can't grok it after reading it several times. Wouldn't a better flow be:

```
  <span ng-if="thread.total_message_count > 1">
    ... handle all the second-last message display stuff here ...
  </span>

   ...handle all the last-message display stuff here...
```